Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM alpine:3.11 AS base
+FROM alpine:3.12 AS base
```

Comment:
[bigfiles] This file, one of the 3 largest in the project, increased in size to 231 lines.  The total size of those files is now 513 lines (target: 455).  Is this file complex enough to refactor?